### ISSUE TITLE
Record Video

### ISSUE THREAD

**Issue by catakkio** [cordova-plugin-camera-preview/cordova-plugin-camera-preview#239]
It's possible to record video during the preview ?

**Comment by mark-ellul** [comment]
any updates on this question?

**Comment by Gerem** [comment]
I'm waiting for this too.

**Comment by faustoct** [comment]
upvoted . it's a must have !

**Comment by mikewagz** [comment]
I would also like to see this.

**Comment by rrundle** [comment]
Have you guys found an alternative solution? I need to deliver a photo booth app that needs this functionality.

**Comment by phonic** [comment]
+1 !

**Comment by HartHerbert** [comment]
+1

**Comment by mikkelsjolin** [comment]
+10 🤠

**Comment by mark-ellul** [comment]
@rrundle I have been looking into the media-capture plugin (https://github.com/apache/cordova-plugin-media-capture)  it allows for both however you have to choose Video, Audio or Photo in the code

**Comment by phonic** [comment]
@mark-ellul yeah, clear, but thats the plain interface without any overlay possibilities.

**Comment by mark-ellul** [comment]
@phonic yes its a bit of a bummer, I really wanted the ability to switch between recording an image or video on the camera view itself, which is not possible without overlays.

**Comment by rrundle** [comment]
@mark-ellul yeah that's probably the best possible alternative at the moment. Having the ability to overlay and take video automatically are pretty nice features of the preview plugin. 

Thanks for the response.

**Comment by AdamAtri** [comment]
+1 - would really like to see this happen

**Comment by andrewoodleyjr** [comment]
+1

**Comment by biga816** [comment]
+1

**Comment by Avishaidev** [comment]
+1 Any solution for this?
I want to create camera preview like on Instegram app.

**Comment by westonganger** [comment]
@vipulsharma144 has submitted PR #587 which completes the work done from @icetec-solutions implementation. I would appreciate if anyone has any comments on that PR.

**Comment by westonganger** [comment]
Video recording functionality for Android has been implemented in master via #587
